create-release/action.yml

```diff
@@ -0,0 +1,53 @@
+name: Create GitHub release
+description: Mostly replicate actions/create-release (which is now unmaintained)
```

Comment:
@dpmatthews This config file is derived from this MIT licenced source - https://github.com/actions/create-release/blob/main/action.yml

Are we good to go with an acknowledgement/license?

https://github.com/actions/create-release/blob/4c11c9fe1dcd9636620a16455165783b20fc7ea0/LICENSE#L13-L14

Reply:
Like this?
```suggestion
# Copyright (c) 2018 GitHub, Inc. and contributors
# Copyright (c) 2023 NIWA & British Crown (Met Office) & Contributors
#
# Permission is hereby granted, free of charge, to any person obtaining a copy
# of this software and associated documentation files (the "Software"), to deal
# in the Software without restriction, including without limitation the rights
# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
# copies of the Software, and to permit persons to whom the Software is
# furnished to do so, subject to the following conditions:
#
# The above copyright notice and this permission notice shall be included in
# all copies or substantial portions of the Software.

name: Create GitHub release
description: Mostly replicate actions/create-release (which is now unmaintained)
```

Reply:
Looks fine to me